README_first.ipynb

```diff
@@ -1,60 +1,160 @@
 {
```

Comment:
<p>I think you can remove the first line: "Here you can learn how to use Hugging Face checkpoints directly for inference."</p><p>I would put this box above the finetuning.</p><blockquote>Our NLP notebooks show how to run inference with only 2 lines of code!</blockquote><p>I think 2 is a bit optimistic :)</p><p> </p><p>Maybe rephrase this box to something like:</p><p> </p><p>See how to solve NLP tasks on the IPU in 5 lines of code, and turn it into an app in 10 lines in our NLP notebooks:</p><p> </p><pre class="ql-syntax" spellcheck="false">    natural-language-processing/ 
        └───sentiment_analysis.ipynb 
        └───name-entity-extraction.ipynb 
</pre><p>Use the sentiment-analysis pipeline to quickly evaluate pre-trained models and check out how to use Gradio with the IPU to prototype a web application doing fast token classification in our name entity extraction notebook.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>